datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -222,6 +225,7 @@ export class ChartEntity implements Entity<Chart> {
             type: EntityType.Chart,
             icon: entity?.platform?.properties?.logoUrl || undefined,
             platform: entity?.platform,
+            subtype: entity?.subTypes?.typeNames?.[0] || undefined,
```

Comment:
bit weird that the "t' in this is lower cased

Reply:
is that intentional?

Reply:
this is existing code. its intentional

Reply:
https://github.com/datahub-project/datahub/blob/399e032dfa2b4bf87b7b406e7b009e34e99a1003/datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx#L329